temba/orgs/migrations/0074_remove_topup_expires_on.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.10 on 2021-06-28 13:33
+
+from django.db import migrations, models
+from django.utils.timezone import now as tz_now, timedelta
+
+
+def year_from_now():
+    return tz_now() + timedelta(days=365)
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("orgs", "0073_merge_20201214_1542"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="topup",
+            name="expires_on",
+            field=models.DateTimeField(
+                default=year_from_now,
+                help_text="The date that this top up will expire",
+                verbose_name="Expiration Date",
+            ),
+        ),
+        migrations.RemoveField(
```

Comment:
This field is also used on Mailroom (https://github.com/nyaruka/mailroom/blob/72657f3f3a146028f6b0955cea08d313c462f028/core/models/topups.go#L127), so I think we should keep it on the code to make it simple on the next merging processes. So I think we only should not include it on the calc of the date expiration.